deploy/kubernetes/helm/che/custom-charts/che-devfile-registry/values.yaml

```diff
@@ -7,6 +7,10 @@
 # SPDX-License-Identifier: EPL-2.0
 #
 
+global:
+  ingress:
+    class: "nginx"
```

Comment:
I am not sure if it is needed since the field is introduced in deploy/kubernetes/helm/che/values.yaml

Reply:
I see, that's make sense. 

1. I believe for custom charts it is better to place all values under corresponding section like this:
```
cheDevfileRegistry:
  ingress:
    class: "nginx"
...
```

2. Missed values for `che-plugin-registry` [1] and `che-keycloak` [2]
[1] https://github.com/eclipse/che/blob/pr-enable-to-customize-ingress-class/deploy/kubernetes/helm/che/custom-charts/che-plugin-registry/values.yaml#L10
[2] https://github.com/eclipse/che/blob/pr-enable-to-customize-ingress-class/deploy/kubernetes/helm/che/custom-charts/che-keycloak/values.yaml#L14

WDYT ?

Reply:
Nice catch. My bad. It should be added to `che-plugin-registry` and `che-keycloak` also.
I added them and force-pushed.